src/scripts/choices.js

```diff
@@ -144,6 +144,7 @@ class Choices {
         classNames: this.config.classNames,
         template: data => this._templates.option(data),
       });
+      this._presetOptions = this.passedElement.options;
```

Comment:
Could this be moved above line 186 please? That's where all the "preset" variables are defined

Reply:
Thanks! I have fixed.